includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -67,44 +67,66 @@ public function plugin_information( $args ) {
 	/**
 	 * Attempt to activate a plugin licence.
 	 *
-	 * @param array|string $args
-	 * @return boolean|string JSON response or false if failed.
+	 * @param array $args The arguments to pass to the API.
+	 * @return array|false JSON response or false if failed.
 	 */
 	public function activate( $args ) {
-		$args            = wp_parse_args( $args );
-		$args['wc-api']  = 'wp_plugin_licencing_activation_api';
-		$args['request'] = 'activate';
-		$response        = $this->request( $args, true );
-		if ( false === $response ) {
+		$args         = wp_parse_args( $args );
+		$product_slug = $args['api_product_id'];
+		$response     = $this->bulk_activate( $args['licence_key'], [ $product_slug ] );
+		if ( false === $response || ! array_key_exists( $product_slug, $response ) ) {
 			return false;
 		}
-		return $response;
+		$item = $response[ $product_slug ];
+		// Add keys used previously for backwards compatibility.
+		$item['activated'] = $item['success'];
+		$item['error']     = $item['error_message'];
+		unset( $item['success'] );
+		unset( $item['error_message'] );
+		return $item;
+	}
+
+	/**
+	 * Attempt to activate multiple WPJM products with a single licence key.
+	 *
+	 * @param string $licence_key The licence key to activate.
+	 * @param array  $product_slugs The slugs of the products to activate.
+	 * @return array|false The response, or false if the request failed.
+	 */
+	public function bulk_activate( $licence_key, $product_slugs ) {
+		return $this->request_endpoint(
+			'wp-json/wpjmcom-licensing/v1/activate',
+			[
+				'method' => 'POST',
+				'body'   => [
+					'site_url'      => $this->get_site_url(),
+					'license_key'   => $licence_key,
+					'product_slugs' => $product_slugs,
+				],
```

Comment:
We need to `json_encode` the body.

Reply:
Nice catch! Fixed in 8a07a35917f33fe6ab617c84d0c251365e4f9cb1 (but using `wp_json_encode` instead), thanks!